expfmt/decode_test.go

```diff
@@ -258,6 +258,65 @@ func TestProtoDecoder(t *testing.T) {
 				},
 			},
 		},
+		{
+			in: "\u007f\n\x1drequest_duration_microseconds\x12\x15The response latency.\x18\x04\"E:C\b\x85\x15\x11\xcd\xcc\xccL\x8f\xcb:A\x1a\v\b{\x11\x00\x00\x00\x00\x00\x00Y@\x1a\f\b\x9c\x03\x11\x00\x00\x00\x00\x00\x00^@\x1a\f\b\xd0\x04\x11\x00\x00\x00\x00\x00\x00b@\x1a\f\b\xf4\v\x11\x9a\x99\x99\x99\x99\x99e@",
```

Comment:
@beorn7 Looking at the spec, we just say "whitespace" rather than tabs and spaces, so we should accept other whitespace characters. Does this seem right to you?

Reply:
It's probably safer to make the spec more precise/restrictive by explicitly only allow spaces and tabs (as it is implemented right now in `expfmt`). Interpreting whitespace widely would include all kind of line-break characters, which we could definitely not allow. But even for non-line-breaking whitespace, there are a _lot_ of whitespace characters in Unicode, which would make parsing much harder if we needed to check for all of them. See https://en.wikipedia.org/wiki/Whitespace_character#Definition_and_ambiguity

Reply:
In different news: You commented the line with the raw protobuf. Where is whitespace involved in that one? Perhaps I'm missing your point.

Reply:
Brainfart on my part, I though this was an exotic text format.

Reply:
But still a good point that we mean “tab and space” when we say "whitespace”. We should clarify (or wait for OpenMetrics to clarify...).

Reply:
OM has only the space character, probably not worth clarifying for Prometheus format at this point.